### ISSUE TITLE
Improvements needed for vApp resource

### ISSUE THREAD

**Issue by vbauzysvmware** [terraform-providers/terraform-provider-vcd#379]
* accept_all_eulas and power_on fields isn't populated after import. We now that accept_all_eulas  cannot be read and power_on is not read because it is not reliable, and is replaced by two more properties. We don't mentioned that documentation.
* after datasource is read - only less than half fields is available

**Comment by dataclouder** [comment]
When we developed importing and data source for vApp, we decided to deprecate all properties that refer to implicit creation of a VM, such as `accept_all_eulas`, `ip`, , and we did not invest time into importing any of these properties into the state. Same goes for data source. Only non-deprecated properties are shown.

**Comment by vbauzysvmware** [comment]
Thank you for update, you are right. Yet in data source documentation we have:
```
## Attribute reference

* `href` - The vApp Hyper Reference
* `metadata` -  Key value map of metadata to assign to this vApp. Key and value can be any string. 
* `power_on` -  A boolean value stating if this vApp should be powered on. Default is `true`
* `guest_properties` -  Key value map of vApp guest properties.
* `status` -  The vApp status as a numeric code
* `status_text` -  The vApp status as text.
```

**Comment by lvirbalas** [comment]
I see, so `power_on` is deprecated but the docs tell it's readable, thanks, nice catch.

**Comment by dataclouder** [comment]
Fixed in 2.6.0 (PR #386)
